.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -39,6 +39,8 @@ Please mark the relevant option(s):
 ## Testing
 
 - [ ] `make develop_test`; if any code changes were made
+- [ ] E2E tests on LocalNet
+- [ ] E2E tests on DevNet
```

Comment:
I'm all about it.

To your second line, we do have docs for DevNet, but I didn't know if it was good to publish them? Just because it links to a lot of internal infra.

Reply:
1. + 💯  to inserting copy pasta commands whenever possible

2. Never hesitate to link to internal notion pages.
2.1 Eventually, we want to make as much of it publically accessible as possible so it's an inline reminds
2.2 It'll help us (the core team and the rest of the eng org) with development
2.3 Worst case, someone sees a link they can't access: no harm
2.4 If an external contributor does try to access it and they can't, we can either help them, tell them to skip it, and see that they're actually putting in the diligence to test everything in detail